2018/20180218.md

```diff
@@ -3,6 +3,14 @@
 ## Community Meeting Summary
 
 
```

Comment:
Maybe we should have a `# Kubernetes Community` section, where KubeVirt is one community?